catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/filter/VisitableXmlElementImplTest.java

```diff
@@ -128,8 +128,10 @@ public void testBinaryComparisonOpType() {
 
   @Test
   public void testPropertyIsLikeType() {
-    parameterizedMappingVerification(
-        PropertyIsLikeType.class, (v) -> verify(v).visitPropertyIsLikeType(eq(node)));
+    PropertyIsLikeType propertyIsLikeType = new PropertyIsLikeType();
+    propertyIsLikeType.setMatchCase(true);
+    propertyIsLikeTypeVerification(
+        propertyIsLikeType, (v) -> verify(v).visitPropertyIsLikeType(eq(node)));
```

Comment:
❗️ We should probably have another test for `setMatchCase(false)` and verify the following: 
```
(v) -> verify(v).visitPropertyIsILikeType(eq(node)));
```

Reply:
done